packages/world/src/systems/MineSystem.sol

```diff
@@ -174,6 +174,13 @@ contract MineSystem is System {
       return NatureLib.growSeed(coord, mined, minedType);
     }
 
+    if (Mass._get(mined) == 0) {
```

Comment:
hmm this function has a bunch of early returns, so this wouldnt set the mass for eg for seeds right?

Reply:
yeah, I did it here as for those we know that it was already set.. but maybe it would be more correct to do it at the beginning

Reply:
yea I think if the idea is that this is a general solution catch-all for all missing mass, then it should be at the beginning imo